L/libcxxwrap_julia/build_tarballs.jl

```diff
@@ -59,4 +59,4 @@ dependencies = [
 
 # Build the tarballs, and possibly a `build.jl` as well.
 build_tarballs(ARGS, name, version, sources, script, platforms, products, dependencies;
-    preferred_gcc_version = v"7", julia_compat = "~$(julia_version.major).$(julia_version.minor)")
+    preferred_gcc_version = v"7", julia_compat = "^$(julia_version.major).$(julia_version.minor)")
```

Comment:
Why this change?  Doesn't the caret mean compatible with `[1.5, 2.0)`?  Is this what you want?

Reply:
https://github.com/JuliaPackaging/Yggdrasil/pull/2158#issuecomment-731643526

Reply:
Oh, well, if this actually works with julia v1.6 that's good :smile:

Reply:
yes, it works in Julia 1.6 (at least on a mac, but it should work everywhere)

Reply:
(famous last words 😝 )